src/js/wizards/Sell/1_PaymentMethods/components/SellerPaymentMethod.jsx

```diff
@@ -3,23 +3,35 @@ import {ButtonGroup} from 'reactstrap';
 import PropTypes from 'prop-types';
 
 import CheckButton from '../../../../ui/CheckButton';
+import { POPULAR_PAYMENT_METHODS, PAYMENT_METHODS } from '../../../../features/metadata/constants';
 
 class SellerPaymentMethod extends Component {
   togglePaymentMethod(selectedMethod) {
-    this.props.togglePaymentMethod(selectedMethod);
+    this.props.togglePaymentMethod(parseInt(selectedMethod, 10));
   }
 
   render() {
     return (
       <React.Fragment>
-        <h2>Payment methods that you want to accept</h2>
+        <h2 className="mb-4">Payment methods that you want to accept</h2>
+        <span className="text-muted text-small">Popular</span>
+        <ButtonGroup vertical className="w-100">
+          {POPULAR_PAYMENT_METHODS.map((idx) => (
+            <CheckButton active={this.props.selectedMethods.indexOf(idx) > -1}
+                         key={'method-' + idx} isCheckBox
+                         onClick={(_e) => this.togglePaymentMethod(idx)}>
+              {PAYMENT_METHODS[idx]}
+            </CheckButton>
+          ))}
+        </ButtonGroup>
 
-        <ButtonGroup vertical className="w-100 mt-3">
-          {this.props.methods.map((asset, idx) => (
+        <span className="text-muted text-small mt-3">All payment methods (A-Z)</span>
+        <ButtonGroup vertical className="w-100">
+          {Object.keys(PAYMENT_METHODS).map(x => parseInt(x, 10)).filter(x => POPULAR_PAYMENT_METHODS.indexOf(x) === -1).map((idx) => (
```

Comment:
Again, I think the first `map` is not really needed

Reply:
Worst case you can do `.filter(x => POPULAR_PAYMENT_METHODS.indexOf(parseInt(x, 10)) === -1)`
instead, that way you don't loop again